bootstrap/config/config.go

```diff
@@ -957,10 +947,45 @@ func (cp *Processor) applyWritableUpdates(serviceConfig interfaces.Configuration
 		lc.Errorf("failed to deep copy insecure secrets: %v", err)
 	}
 
-	if err := utils.MergeValues(serviceConfig.GetWritablePtr(), raw); err != nil {
-		lc.Errorf("failed to apply Writable change to service configuration: %v", err)
+	serviceConfigCopy, err := copyConfigurationStruct(serviceConfig)
```

Comment:
this line cause unit test failure in https://jenkins.edgexfoundry.org/job/edgexfoundry/job/go-mod-bootstrap/job/PR-878/1/display/redirect